src/pds/registry/utils/geostac/create_lola_pds4.py

```diff
@@ -82,15 +83,16 @@ def create_product_external(item):
     """
     # create env
     env = Environment()
-
     with importlib.resources.open_text(templates, "product-external-template.xml") as io:
         template_text = io.read()
         template = env.from_string(template_text)
 
         item_title = item["assets"]["data"]["title"]
 
         last_slash_i = item["assets"]["data"]["href"].rfind("/")
-        file = item["assets"]["data"]["href"][last_slash_i + 1:]
+        file = "data/" +item["assets"]["data"]["href"][last_slash_i + 1:]
+        print(f'file is on {item["assets"]["data"]["href"]},fake file is on {file}')
```

Comment:
Should this be using `logging.info` instead? If not, should it be using `print(…, file=sys.stderr)` instead?